core/templates/pages/exploration-player-page/exploration-player-page.module.ts

```diff
@@ -68,6 +70,7 @@ import { MyHammerConfig, toastrConfig } from 'pages/oppia-root/app.module';
     LearnerLocalNavComponent,
     FeedbackPopupComponent,
     LearnerViewInfoComponent,
+    RatingsAndRecommendationsComponent,
```

Comment:
Any specific reason for not having separate rating and recommendation components? (They looks pretty independent view to me)

Reply:
Not sure, but this component is not that big that it requires further sub-division.